### PR TITLE
Add onesignal push notification

### PR DESCRIPTION
**Pull request by kellybinary**

### CODE REVIEW

File: src/javascript/lib/onesignal.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        this.OneSignal.push(['init', this.options]);
+    }
+
+    /*
+     * Checks if a user has already accepted push notifications,
+     * then send the user website language and Binary login_id to OneSignal
+     */
+    checkSubscription() {
+        this.OneSignal.push(() => {
+            // if (window.location.hostname !== 'www.binary.com') return; // Use in production
```
Comment (binary-hamedanchi) [inline src/javascript/lib/onesignal.js]: Need to uncomment this line before merge.

Comment (kellybinary) [inline src/javascript/lib/onesignal.js]: @binary-hamedanchi Thanks for catching that, will uncomment it.

File: src/javascript/lib/onesignal.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    /*
+     * Initializes OneSignal SDK
+     * Notes:
+     *  - do not call 'init' twice
+     *  - 'init' call is required before any function can be used
+     *  - log level can be set to 'trace/debug/info/warn/error' or 'silent' to disable all but in production
+     *     - refer https://www.npmjs.com/package/loglevel
+     */
+    configure() {
+        this.OneSignal.log.setLevel('silent'); // Disable all error logging
```
Comment (kellybinary) [inline src/javascript/lib/onesignal.js]: @binary-hamedanchi error logs can still be seen in production. is this ok?

File: src/sass/onesignal.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/* Override default OneSignal popover css */
+#onesignal-popover-container #onesignal-popover-dialog {
```
Comment (kellybinary) [inline src/sass/onesignal.scss]: @binary-hamedanchi this file is no longer useful, remove this entirely?

Comment (binary-hamedanchi) [inline src/sass/onesignal.scss]: Yes, please.

**Pull request merged**